src/dune_threaded_console/dune_threaded_console.ml

```diff
@@ -62,7 +62,10 @@ let make (module Base : S) : (module Dune_console.Backend) =
       Base.start ();
       Dune_engine.Scheduler.spawn_thread @@ fun () ->
       if not Sys.win32 then
-        ignore (Unix.sigprocmask SIG_UNBLOCK [ Signal.to_int Winch ] : int list);
+        ignore
+          (Unix.sigprocmask SIG_UNBLOCK
+             (List.map ~f:Signal.to_int [ Winch; Cont; Tstp ])
```

Comment:
Can we avoid writing this list twice?

Reply:
I don't mind, but it would have to go somewhere odd like `Dune_util`. As we don't want the console to depend on the engine.